### ISSUE TITLE
The problem of IRM module input and model stability

### ISSUE THREAD

**Issue by Ethevliu** [Show-han/PET-Reconstruction#12]
Hello, I have a few questions for you about this paper. First of all, it is the input of IRM module. The model diagram in this paper shows that the input of IRM module is LPET image, but the input described in this paper is IP image, and the input in the code is the residual image of HR and IP image. So I want to know what are the inputs to the IRM model?

**Comment by Ethevliu** [comment]
I tested the node model you provided, and found that the average SSIM was 0.962, which was slightly different from 0.975 in the paper, because there was a code error in the calculation of SSIM in the our_paper_metry. py file you provided before. I changed it to cur_ssim = ssim(SPETimg, EPETimg, data_range=dr,multi_channel=1). Will this change affect the calculation of SSIM?
![image](https://github.com/user-attachments/assets/a8e20652-59f3-418d-8f0f-5e2f2c0b8cc3)

**Comment by Show-han** [comment]
For the input, the provided code is definitely the most accurate. Please understand it according to the code. As for the calculation of SSIM, please also refer to the specific code for understanding.

**Comment by liudelong123123** [comment]
Can you provide the code for other model training in the paper? If possible, thank you very much.

**Comment by liudelong123123** [comment]
Can you provide the code for other model training in the paper? If possible, thank you very much.

**Comment by Ethevliu** [comment]
Can you provide code for other model training that compares to the metrics in the paper? If possible, thank you very much.

**Comment by Show-han** [comment]
Unfortunately, since a long time has passed, and I have left the original lab and moved on to a different research direction, I no longer have the code for the comparison experiments. You might want to try reaching out to the authors of the comparison paper to obtain the code.

**Comment by XiaJiayu22** [comment]
Did you run this result using inferrence.py?

**Comment by XiaJiayu22** [comment]
I obtained the same results as you. Did you finally obtain the results in the paper

**Comment by Ethevliu** [comment]
> I obtained the same results as you. Did you finally obtain the results in the paper
 No, the result measured by its best node is the above result. However, the process involves random procedures. You can test it several times, but the results won't vary much.
